### PR TITLE
[KOGITO-3915] Updating examples for OpenApi Client feature

### PR DESCRIPTION
**Pull request by ricardozanini**
Signed-off-by: Ricardo Zanini <zanini@redhat.com>

Updating examples to reflect changes on https://github.com/kiegroup/kogito-runtimes/pull/1014

Many thanks for submitting your Pull Request :heart:! 

Please make sure that your PR meets the following requirements:

**WARNING! Please make sure you are opening your PR against `master` branch!**

- [x] You have read the [contributors guide](https://github.com/kiegroup/kogito-runtimes#contributing-to-kogito)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting master: `[0.9.x] KOGITO-XYZ Subject`
- [x] Pull Request contains link to the JIRA issue
- [x] Pull Request contains link to any dependent or related Pull Request
- [x] Pull Request contains description of the issue
- [x] Pull Request does not include fixes for issues other than the main ticket

### CODE REVIEW

**Review by tsurdilo — APPROVED** [review]

File: pom.xml [hunk 1]
```diff
@@ -49,6 +49,8 @@
     <version.org.optaplanner>8.4.0-SNAPSHOT</version.org.optaplanner>
     <!-- pmml -->
     <version.org.jpmml>1.5.1</version.org.jpmml>
+    <!-- openapi client -->
+    <version.org.openapitools.jackson.databind.nullable>0.2.0</version.org.openapitools.jackson.databind.nullable>
```
Comment (cristianonicolai) [inline pom.xml]: we shouldnt need to repeat is here as already defined in paret build pom?

File: serverless-workflow-service-calls-quarkus/pom.xml [hunk 2]
```diff
@@ -60,6 +60,26 @@
...
+      <artifactId>resteasy-client</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.resteasy</groupId>
+      <artifactId>resteasy-multipart-provider</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.openapitools</groupId>
+      <artifactId>jackson-databind-nullable</artifactId>
+    </dependency>
```
Comment (danielezonca) [inline serverless-workflow-service-calls-quarkus/pom.xml]: Why these dependencies?
I don't expect user to add them explicitly. If they are required in this scenario I would prefer to have them as transitive dependencies of `kogito-openapi-workitem`

File: serverless-workflow-service-calls-springboot/pom.xml [hunk 3]
```diff
@@ -55,6 +55,29 @@
...
+      <artifactId>resteasy-multipart-provider</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.jboss.resteasy</groupId>
+      <artifactId>resteasy-jackson2-provider</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.openapitools</groupId>
+      <artifactId>jackson-databind-nullable</artifactId>
+    </dependency>
```
Comment (danielezonca) [inline serverless-workflow-service-calls-springboot/pom.xml]: Same as above

File: serverless-workflow-service-calls-quarkus/pom.xml (hunk 2 above)
Comment (ricardozanini) [reply serverless-workflow-service-calls-quarkus/pom.xml]: Duh, of course. I'll do that.

File: pom.xml (hunk 1 above)
Comment (ricardozanini) [reply pom.xml]: Yeah, I need to remove it. Slipped from my review. Too many files :worried:

**Review by evacchi — APPROVED** [review]

**Comment by ricardozanini** [comment]
jenkins retest this please

**Comment by ricardozanini** [comment]
Can be merged after https://github.com/kiegroup/kogito-runtimes/pull/1014

**Comment by ricardozanini** [comment]
kiegroup/kogito-runtimes#1014 is green. Everything ready to be merged.

**Pull request merged**